.github/workflows/licensecheck.yml

```diff
@@ -1,113 +1,22 @@
-# https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions
-name: License check
+# This workflow will check for Maven projects if the licenses of all (transitive) dependencies are vetted.
+
+name: License vetting status check
 
 on:
   push:
-    branches-ignore: # build all branches except:
-      - 'dependabot/**' # prevent GHA triggered twice (once for commit to the branch and once for opening/syncing the PR)
-    tags-ignore: # don't build tags
-    - '**'
+    branches: 
+      - 'main'
```

Comment:
I don't think the license check should only run on the main branch. PRs should only be merged once the license check was successful.